website/src/components/canvas/CanvasView.jsx

```diff
@@ -40,13 +45,21 @@ class CanvasView extends React.Component {
    */
   componentDidMount() {
     firebase.database().ref('/test').once('value').then((elemListSnap) => {
-      this.props.dispatch(initPositions(elemListSnap.val()));
+      this.props.dispatch(initElements(elemListSnap.val()));
     });
 
     firebase.database().ref('/test').on('child_changed', (elemSnap) => {
```

Comment:
might be useful to see what has changed and if only one dispatch needs to be called. Right now, there will be three dispatches every time another user moves an object in FB